modular_skyrat/modules/nanotrasen_naval_command/code/outfits.dm

```diff
@@ -0,0 +1,197 @@
+/datum/outfit/centcom/naval
+
+/datum/outfit/centcom/naval/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+
+	var/obj/item/card/id/W = H.wear_id
+	W.registered_name = H.real_name
+	W.update_label()
+	W.update_icon()
+	..()
```

Comment:
Single letter vars are so 2012, you're better than this.

Reply:
It's copy pasted.